runtime/parachains/src/paras.rs

```diff
@@ -1726,7 +1801,10 @@ impl<T: Config> Pallet<T> {
 	fn decrease_code_ref(code_hash: &ValidationCodeHash) -> Weight {
 		let mut weight = T::DbWeight::get().reads(1);
 		let refs = <Self as Store>::CodeByHashRefs::get(code_hash);
-		debug_assert!(refs != 0);
```

Comment:
This is removed just to pass an extra paranoid test: `increase_code_ref_doesnt_have_allergy_on_add_trusted_validation_code`